frontend/pages/browse.js

```diff
@@ -213,7 +238,12 @@ async function getMembers(page, token, urlEnding) {
 
 async function getDataFromServer({ type, page, token, urlEnding }) {
   let url = `${process.env.API_URL}/api/${type}/?page=${page}`;
-  if (urlEnding) url += urlEnding;
+
+  // Handle query params as well
+  if (urlEnding) {
+    // &category=Lowering%20food%20waste&
+    url += urlEnding;
+  }
```

Comment:
For single line `if` I generally just add two braces for readability. Happy to back out if undesired.